services/contributions.js

```diff
@@ -0,0 +1,96 @@
+const githubService = require('../services/githubService')
+const tasks = require('../models/tasks')
+
+/**
+ * Get the  contributions of the user
+ * @param {string} username
+ */
+
+async function getUserContributions (username) {
```

Comment:
Not using arrow function here?

Reply:
Okay, I have updated it to arrow function :smile: